stream.go

```diff
@@ -110,7 +110,8 @@ type ClientStream interface {
 	// It should not be called until after Header or RecvMsg has returned. Once
 	// called, subsequent client-side retries are disabled.
 	Context() context.Context
-	// SendMsg is generally called by generated code. On error, SendMsg aborts
+	// SendMsg is generally called by generated code. On success, SendMsg returns
```

Comment:
Based on the last comment from issue, we just need to an addition sentence indicating that, in case of ClientStreams=false for StreamDesc https://github.com/grpc/grpc-go/blob/56df169480cdb4928a24a50b5289f909f0d81ba7/stream.go#L75, nil is returned unconditionally.